sample/build.gradle

```diff
@@ -115,6 +115,7 @@ if (isCi) {
           String outputRules = outputProguardRules.getAsFile().get().text
           String expectedRules = file("expectedRules.pro").text
           if (outputRules != expectedRules) {
+            println "Invalid rules: " + outputRules
```

Comment:
nit: let's move this to the message in the exception instead

Reply:
Done in 1c87a67